components/skus/browser/rs/lib/src/storage/kv.rs

```diff
@@ -180,6 +260,49 @@ where
         return Ok(credentials);
     }
 
+    #[instrument]
+    async fn upsert_time_limited_v2_item_creds(
+        &self,
+        item_id: &str,
+        creds: Vec<Token>,
+    ) -> Result<(), InternalError> {
+        let mut store = self.get_store()?;
+        let mut state: KVState = store.get_state()?;
+
+        if state.credentials.is_none() {
+            state.credentials = Some(CredentialsState::default());
+        }
+
+        if let Some(credentials) = state.credentials.as_mut() {
+            // check and see if we already have this item initialized
+            if let Some(item_credentials) = credentials.items.get_mut(item_id) {
+                // don't overwrite the existing creds, we just want to add the new blinded
```

Comment:
this looks like overwriting?

Reply:
correcting the comment to make more sense after we renamed creds-> unblinded_creds and tokens->creds